src/modules/error/components/__tests__/Failure.spec.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import Failure from '../Failure'
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import { brightTheme } from '../../../theme/constants/theme'
+
+describe('Failure', () => {
+  it('should render a retry button if tryAgain is passed', () => {
+    const { getByTestId } = render(<Failure theme={brightTheme} error={new Error()} tryAgain={() => {}}
+                                            t={key => key} />)
+
+    expect(getByTestId('button-tryAgain')).not.toBeNull()
```

Comment:
wouldn't `expect(xyz).toBeTruthy()` be a safer way? Not sure wether getByTestId returns null or undefined or false if it's unsuccessfull... Also you seem to use .toBeTruthy in getByText below :thinking: